lib/dry/schema/path.rb

```diff
@@ -100,6 +100,8 @@ def last
       def same_root?(other)
         root.equal?(other.root)
       end
+
+      EMPTY = new([]).freeze
```

Comment:
Not ideal placement, but dunno where else should I put it.

Reply:
Super minor but maybe `EMPTY = new(EMPTY_ARRAY).freeze`?

Reply:
done